### ISSUE TITLE
Bonus Features Folder

### ISSUE THREAD

**Issue by trumblejoe** [jellyfin/jellyfin#1088]
Presently Specials are placed under the season-level folder Specials, it would be beneficial to see this functionality extended out to 'Bonus Features' at the Season Level through a Bonus Features directory.

This would enable encompassing all junk drawer items which are not necessarily listed in the TVDB listings under specials. Cast Interviews, OP/ED files, Page Scans, Concept Art files, etc.

**Comment by EraYaN** [comment]
Does the "extras" folder work for TV Shows?

**Comment by trumblejoe** [comment]
So this is becoming interesting...I created a tv show library with a single tv series in it.

Dir Structure:
TV Show/Season 1
TV Show/Extras

It scanned Season 1 and it's file correctly, it also created a "Specials" folder, with nothing inside of it (probably because the file-types are being ignored by the scanner, I used .bmp and .txt files in the Extras directory).

I'm thinking it's picking up the Extras folder but because of my library setting for "Special Season Display Name" it's naming it 'Specials'. I believe it sees any folder not explicitly named, 'Season XX' as a folder containing specials.
